### DISCUSSION TITLE
Added a new Contract Connection

### DISCUSSION THREAD

**Discussion by Redavatar1** [DarwinsBuddy/WienerNetzeSmartmeter#126] [Q&A] [answered]
I recently added a second Connection to my Smart-Meter portal, since then i have not gotten any data from the plugin. I reinstalled it and I got it working again, just with the wrong device. Is there a way to import both smart meters or can i pick witch one I want to import into homeassistant?

**Comment by DarwinsBuddy** [comment]
a little late to the party, as I'm not frequently checking the Discussion. do you still have this issue?

**Comment by DarwinsBuddy** [comment, marked as answer]
you can configure your smartmeters manually via `configuration.yaml` for a specific set up. usually the smartmeters should be picked up by the integration automatically when you configure it via the UI. If you still don't see your smart meter(s) show up, please follow these steps

[How can I add a new Smartmeter Type](https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/wiki/FAQs#how-can-i-add-a-new-smartmeter-type)
